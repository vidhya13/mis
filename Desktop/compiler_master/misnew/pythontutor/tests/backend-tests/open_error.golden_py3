{
  "code": "for line in open(\"/etc/passwd\"):\n    print(line)\n\n", 
  "trace": [
    {
      "ordered_globals": [], 
      "stdout": "", 
      "func_name": "<module>", 
      "stack_to_render": [], 
      "globals": {}, 
      "heap": {}, 
      "line": 1, 
      "event": "step_line"
    }, 
    {
      "ordered_globals": [], 
      "stdout": "", 
      "exception_msg": "Exception: open() is not supported by Python Tutor.\nInstead use io.StringIO() to simulate a file.\nHere is an example: http://goo.gl/uNvBGl", 
      "func_name": "<module>", 
      "stack_to_render": [], 
      "globals": {}, 
      "heap": {}, 
      "line": 1, 
      "event": "exception"
    }
  ]
}
