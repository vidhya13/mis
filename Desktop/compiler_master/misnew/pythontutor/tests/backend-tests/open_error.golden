{
  "code": "for line in open(\"/etc/passwd\"):\n    print(line)\n\n", 
  "trace": [
    {
      "ordered_globals": [], 
      "stdout": "", 
      "func_name": "<module>", 
      "stack_to_render": [], 
      "globals": {}, 
      "heap": {}, 
      "line": 1, 
      "event": "step_line"
    }, 
    {
      "ordered_globals": [], 
      "stdout": "", 
      "exception_msg": "Exception: open() is not supported by Python Tutor.\nInstead use StringIO.StringIO() to simulate a file.\nHere is an example: http://goo.gl/Q9xQ4p", 
      "func_name": "<module>", 
      "stack_to_render": [], 
      "globals": {}, 
      "heap": {}, 
      "line": 1, 
      "event": "exception"
    }
  ]
}
